packages/console/src/pages/TenantSettings/TenantBasicSettings/index.tsx

```diff
@@ -47,7 +40,7 @@ function TenantBasicSettings() {
   const { show: showModal } = useConfirmModal();
 
   const methods = useForm<TenantSettingsForm>({
-    defaultValues: tenantProfileToForm(currentTenant),
+    defaultValues: { profile: currentTenant },
```

Comment:
Will it lead to a different outcome from the current code?

Reply:
how so? we can `pick` before passing it though

Reply:
discussed offline. was a type issue